utils/constants.go

```diff
@@ -6,9 +6,11 @@ const (
 	SnykToken string = ""
 	AuthToken string = "3e42fa66f65124e6b1266a23431e3d08"
 	Debug     bool   = false
+	Shell     string = "/bin/sh"
 )
 
 // API Constants
 const (
 	APIUsers = "user"
+	APISA    = "api/v2/stack-analyses"
```

Comment:
Usually the api path should start with slash.
```suggestion
	APIUsers = "/user"
	APISA    = "/api/v2/stack-analyses"
```

Also there is no point in having all api endpoints in a dedicated file. These endpoints must be stored as a constant in corresponding implementation.

inside auth/api.go  
inside analyse/api.go

Reply:
Ack. prepended slash to every endpoint